### ISSUE TITLE
blockbreakevent

### ISSUE THREAD

**Issue by arturek1666** [CoolLord22/OtherDrops#28]
**Describe the bug**
The plugin just spams with the error

**To Reproduce**
1. Start the server
2. Block any block
3. After breaking the block there should be error on the console

**Expected behavior**
Massive errors in console like crazy 

**Desktop (please complete the following information):**
 - OS: windows 10
 - Java Version: java 8
 - Server Version: paperspigot 1.13
 - Plugin Version: 3.2.5

**Additional context**
Error log:

```
[21:48:39] [Server thread/ERROR]: Could not pass event BlockBreakEvent to OtherDrops v3.2.5
java.lang.NoSuchMethodError: com.sk89q.worldguard.bukkit.WorldGuardPlugin.getRegionContainer()Lcom/sk89q/worldguard/bukkit/RegionContainer;
	at com.gmail.zariust.otherdrops.event.OccurredEvent.setRegions(OccurredEvent.java:803) ~[?:?]
	at com.gmail.zariust.otherdrops.event.OccurredEvent.<init>(OccurredEvent.java:130) ~[?:?]
	at com.gmail.zariust.otherdrops.listener.OdBlockListener.onBlockBreak(OdBlockListener.java:106) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor868.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.13.jar:git-Paper-126]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.jar:git-Paper-126]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.jar:git-Paper-126]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:527) ~[patched_1.13.jar:git-Paper-126]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:512) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.PlayerInteractManager.breakBlock(PlayerInteractManager.java:297) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.PlayerInteractManager.a(PlayerInteractManager.java:223) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.PlayerConnection.a(PlayerConnection.java:1252) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.PacketPlayInBlockDig.a(SourceFile:40) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.PacketPlayInBlockDig.a(SourceFile:10) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:13) ~[patched_1.13.jar:git-Paper-126]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_144]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_144]
	at net.minecraft.server.v1_13_R1.SystemUtils.a(SourceFile:198) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.MinecraftServer.w(MinecraftServer.java:986) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.DedicatedServer.w(DedicatedServer.java:428) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.MinecraftServer.v(MinecraftServer.java:914) ~[patched_1.13.jar:git-Paper-126]
	at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:812) ~[patched_1.13.jar:git-Paper-126]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_144]
```

**Comment by CoolLord22** [comment]
Hey! This was already fixed in a previous code change, just hasn't been committed to GitHub yet. It has already been reported on Discord and I have already fixed. WIll be released in master 1.13 branch of 3.2.6

**Comment by CoolLord22** [comment]
Please leave issues open until code commits are pulled into master branch. The issue will close automatically :) thank you!
